posthog/migrations/0132_team_test_account_filters.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 3.0.6 on 2021-02-25 17:58
+
+import re
+
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations
+
+from posthog.utils import GenericEmails
```

Comment:
Note: This has the potential to cause broken updates down the line if the import path ever changes.